theme-data/dark/dark-common.json

```diff
@@ -107,7 +107,8 @@
     },
     "button": {
       "border": {
-        "normal": "none"
+        "normal": "@color-transparent",
```

Comment:
@sdule what are your thoughts on this? I saw you added this token around 9 months ago

Reply:
Yes the explanation seems correct, not familiar with how 'none' works but transparent should be fine.

Reply:
@nataliadelmar The color was added 9 months ago but was never used up to this point. Now we set/use a border for `button-primary` and `button-ghost`, because we need to have a border for High Contrast - this makes the incorrect color visible for non-HC themes, so we needed to hide it(in non-HC themes, the border should not be visible).

Reply:
I remembered. "none" is a keyword in the theme_generator. When we assigned the border to "none", we assign the same color of the background to the border. We want to keep that behaviour for the non-HC borders, which are non-existent. 
![image](https://user-images.githubusercontent.com/56999622/174281012-be4be600-4cc8-4670-b39d-6b6571ca5fd3.png)

Reply:
"@color-transparent" wouldn't have the same effect as "none" for the non-HC Themes

Reply:
@nataliadelmar 

> We want to keep that behaviour for the non-HC borders, which are non-existent

The borders were non-existent*.
We need to associate a token now for borders, in order to be able to control the look of these, under High Contrast, like that disabled state you can see in the screenshots.
The regular themed buttons look exactly as these looked before the change.